YIF.Core.Service/Concrete/Services/SuperAdminService.cs

```diff
@@ -462,5 +462,16 @@ public async Task<ResponseApiModel<DescriptionResponseApiModel>> ChangeBannedSta
             }
             return result.Set(new DescriptionResponseApiModel(res), true);
         }
+
+        public async Task<ResponseApiModel<DescriptionResponseApiModel>> GetIoEAdminIdByIoEId(string ioEId) {
+            var result = new ResponseApiModel<DescriptionResponseApiModel>();
+            var admin = await _institutionOfEducationAdminRepository.GetByInstitutionOfEducationId(ioEId);
+
+            if (admin == null) {
+                throw new NotFoundException(_resourceManager.GetString("IoEWasNotFoundOrAdminWasDeleted"));
```

Comment:
why you have one message that describes two different errors?

Reply:
It is because of the validation inside repository method, if it's necessary, I can divide them in two different